src/main/java/com/andreamazzarella/contact_manager/ConsoleRunner.java

```diff
@@ -0,0 +1,36 @@
+package com.andreamazzarella.contact_manager;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class ConsoleRunner {
+    public static void main(String[] args) throws IOException {
+        InMemoryRepository myRepo = new InMemoryRepository();
+
+        initialiseRepository(myRepo);
+
+        TextBasedContactManager contactManager = new TextBasedContactManager(System.in, System.out, myRepo);
+        contactManager.run();
+    }
+
+    private static void initialiseRepository(InMemoryRepository myRepo) {
+        HashMap<String, String> andreasInfo = new HashMap<>();
```

Comment:
In Java, it's best to code with the supertype to the left, and subtype to the rigth, so

`Map<String, Integer> map = new HashMap<>();`